latest_requirements.txt

```diff
@@ -1,5 +1,5 @@
-boto3==1.34.80
-botocore==1.34.80
+boto3==1.34.81
+botocore==1.34.81
```

Comment:
should we remove boto from the list of libraries we're auto checking for and just cap at the next major version?

Reply:
yah, they have very frequent releases